node_modules/col-canvas/lib/poller.js

```diff
@@ -392,6 +416,7 @@ var handleUser = function(ctx, users, canvasUser, callback) {
   // Otherwise we create or update a record for the user
   var defaults = {
     'canvas_course_role': courseRole,
+    'canvas_course_sections': canvasUser.course_sections,
```

Comment:
@johncrossman Should `'canvas_course_sections' === canvasUser.course_sections` be added to the return-early check just above this, so that the function doesn't miss a change in section membership?